android/app/src/main/java/org/ZingoLabs/Zingo/RPCModule.kt

```diff
@@ -357,6 +357,11 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
     fun saveWallet() {
         // Get the encoded wallet file
         val b64encoded = uniffi.rustlib.saveToB64()
```

Comment:
When `saveToB64` returned an Error, like: `Error: Lightclient is not initialized` the App saved it as a content of the wallet file. Maybe this happened sometimes but the problem was when this happened the last time that the App saved the wallet. We didn't have any issue from Android, but the problem existed here as well.